packages/legacy-events/EventPluginUtils.js

```diff
@@ -80,20 +80,12 @@ export function executeDispatchesInOrder(event) {
     validateEventDispatches(event);
   }
   if (Array.isArray(dispatchListeners)) {
-    let previousInstance;
     for (let i = 0; i < dispatchListeners.length; i++) {
-      const instance = dispatchInstances[i];
```

Comment:
Can you explain what this code was doing? I'm not sure I understand why we would have the same instance twice, or how that relates to propagation. I see it was added very recently? Is this used by the legacy event system? Seems a bit scary to change it.

Reply:
I think this PR is essentially removing the modified code, and instead forking it for modern system only?

Reply:
This is reverting this file back to what it was before any of the changes from the modern event system were made to it. Like you say, we instead use a forked version instead.